group_by.py

```diff
@@ -0,0 +1,18 @@
+
+def group_by(func: callable, data_base: iter) -> dict:
```

Comment:
callable and iter are not types, they are functions. Change to proper usage - 
https://docs.python.org/3/library/typing.html